README.md

```diff
@@ -36,9 +36,11 @@ To use the script, you will need to follow these steps:
 1. Clone the repository via `git clone https://github.com/yoheinakajima/babyagi.git` and `cd` into the cloned repository.
 2. Install the required packages: `pip install -r requirements.txt`
 3. Copy the .env.example file to .env: `cp .env.example .env`. This is where you will set the following variables.
-4. Set your OpenAI and Pinecone API keys in the OPENAI_API_KEY, OPENAPI_API_MODEL, and PINECONE_API_KEY variables.
-5. Set the Pinecone environment in the PINECONE_ENVIRONMENT variable.
-6. Set the name of the table where the task results will be stored in the TABLE_NAME variable.
+4. Set your OpenAI API keys in the OPENAI_API_KEY and OPENAPI_API_MODEL variables.
+5. Select a vector database type for your context storage by setting the CONTEXT_STORAGE_TYPE variable.
```

Comment:
By default, I think it would be best if no additional configuration was required of the user - Chroma supports this out of the box, see: https://github.com/yoheinakajima/babyagi/pull/72